core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestService.java

```diff
@@ -299,8 +299,8 @@ protected void notifyKitStatusChange(KitRequest kitRequest, KitRequestStatus pri
         if (priorStatus == kitRequest.getStatus()) {
             return;
         }
-        // for now, only notify when the kit is sent
-        if (!KitRequestStatus.SENT.equals(kitRequest.getStatus())) {
+        // only notify when the kit is sent or received
+        if (!KitRequestStatus.SENT.equals(kitRequest.getStatus()) && !KitRequestStatus.RECEIVED.equals(kitRequest.getStatus())) {
```

Comment:
I think a list operation is clearer and more extensible here (possibly pulling the list out into a "EVENTED_STATUSES" constant too)
```suggestion
        if (!List.of(KitRequestStatus.SENT, KitRequestStatus.RECEIVED).contains(kitRequest.getStatus)) {
```